src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -115,13 +100,12 @@ const CoinMachine = ({
 
   const {
     data: periodTokensData,
-    refetch: refetchCurrentPeriodTokensData,
-    startPolling: startPollingCurrentPeriodTokensData,
     stopPolling: stopPollingCurrentPeriodTokensData,
     loading: periodTokensLoading,
   } = useCurrentPeriodTokensQuery({
     variables: { colonyAddress },
     fetchPolicy: 'network-only',
+    pollInterval: 1000,
```

Comment:
One second is waaaay too agressive.

It should be based on blocktime, so about 5 seconds for xdai, and about 20-ish for mainnet

Note: this applies to all three